test/scale-infra/README.md

```diff
@@ -0,0 +1,83 @@
+A scalable Kubernetes-based testbed for KubeStellar Performance Tests
+---------------------------------------------------------------------
+
+<img src="images/ks-scale-test-infra.jpg" width="60%" height="60%" title="ks-scale-test-infra">
+
+### Prerequisites
+- Ansible, with 
+  - the `amazon.aws` collection installed (see, e.g., https://docs.ansible.com/ansible/latest/collections/amazon/aws/ec2_module.html), and
+  - [PyPI package boto3](https://pypi.org/project/boto/) installed for the Python interpreter that Ansible uses on your machine;
+- AWS CLI;
+- AWS access key ID and AWS secret access key (to configure AWS CLI [see](https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-files.html#cli-configure-files-methods))
+- An SSH "keypair" registered with EC2
+
+### Overview
+
+The testbed consists of (from bottom to top):
+- Networking and storage resources from AWS;
+- AWS EC2 instances;
+- Kubernetes cluster(s);
+- Workload(s): Kubestellar Core and WEC components 
+
+
+### Quick Start Deployment Instructions:
+
+1. Deploy the KS control plane hosting infra: 
+
+    ```
+    ./deploy_ks_cp_infra.sh --region us-east-2 --k8s_num_masters 3 --k8s_num_workers 3 --instances_type t2.xlarge   --aws_key_name mykey  --arch x86_64 --ks_release 0.25.0
+    ```
+
+    The above command creates the required AWS infrastructure including a VPC, security groups and EC2 instances. Then, it creates a Kubernetes cluster deployed using Kubeadm. Lastly, it deploys the KubeStellar core components. You can use the flag `--ks_release` to specify the KubeStellar release.
+
+
+2. Create the WEC hosting instances:
+
+    ```
+    ./deploy_wec_infra.sh --region us-east-2 --num_hosts 1 --instance_type t2.2xlarge --aws_key_name  mykey  --arch x86_64
+    ```
+
+    Use the flag `--num_hosts` to specify the number of ec2 instances to be created. 
```

Comment:
I think that it would be best if this sentence and the picture used the same words.

Reply:
Fixed. Replaced the flag name with the following one: `--wecs_hosting_instances`